test/unit/mem/view/CMakeLists.txt

```diff
@@ -59,6 +59,12 @@ SET(_SOURCE_DIR "src/")
 # Add all the source files in all recursive subdirectories and group them accordingly.
 append_recursive_files_add_to_src_group("${_SOURCE_DIR}" "${_SOURCE_DIR}" "cpp" _FILES_SOURCE)
 
+# FIXME: The appveyor CI builds crash with out of memory when building this file.
+IF(ALPAKA_CI)
```

Comment:
this will also exclude it in travis. Maybe just control the one file/test to be not executed on appveyor?

The general CMake way is to set `CTEST_CUSTOM_TESTS_IGNORE` in appveyor *externally* when calling cmake:
https://www.vtk.org/Wiki/CMake/Testing_With_CTest?ref=driverlayer.com/web

Reply:
You are right, this also exludes the file from travis builds. I will fix this.
However, a simple test exclusion is not enough. This file fails to compile, so we have to exclude it from building at all.